### PR TITLE
Hotfix/user drink page drinks page

### PR DESCRIPTION
**Pull request by Disclo5ure**

### CODE REVIEW

**Comment by Disclo5ure** [comment]
Also added onBlur functionality for search input

File: src/components/SearchFilterContainer/SearchFilterContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  useEffect(() => {
+    dispatch(getCategoriesThunk());
+    dispatch(getIngredientsThunk());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(
+      searchDrinksThunk({ drink: name, ingredients: ingredient, category })
+    );
+  }, [ingredient, category, dispatch]);
```
Comment (Redddeve) [inline src/components/SearchFilterContainer/SearchFilterContainer.jsx]: ESLint error
```suggestion
  }, [ingredient, category, dispatch, name]);
```

**Review by Redddeve — CHANGES_REQUESTED** [review]
Fix errors

**Review by Redddeve — APPROVED** [review]
LGTM, waiting for more updates

**Pull request merged**